r/R/dplyr-funcs-datetime.R

```diff
@@ -518,6 +518,28 @@ register_bindings_datetime_parsers <- function() {
 
     coalesce_output <- build_expr("coalesce", args = parse_attempt_expressions)
 
-    build_expr("assume_timezone", coalesce_output, options = list(timezone = tz))
+    if (!is.null(tz)) {
```

Comment:
Please can you add a brief comment about why we need this logic here?

Reply:
Done